modules/main/src/main/scala/MainConfiguration.scala

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2016-2017 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package gem.main
+
+import gem.web.WebConfiguration
+import gem.dao.DatabaseConfiguration
+import gem.telnetd.TelnetdConfiguration
+
+final case class MainConfiguration(
+  database: DatabaseConfiguration,
+  web:      WebConfiguration,
+  telnetd:  TelnetdConfiguration
+)
+
+object MainConfiguration {
+
+  val forTesting: MainConfiguration =
+    apply(
+      DatabaseConfiguration.forTesting,
+      WebConfiguration.forTesting,
```

Comment:
I wish we could coordinate with ocs3 on these details otherwise it will be harder to merge. We already have some configuration classes for Web and other subsystems